### ISSUE TITLE
Openlitespeed restart: sh: usermod: command not found

### ISSUE THREAD

**Issue by franciscopaniskaseker** [litespeedtech/openlitespeed#207]
**My environment:**

```bash
[root@mail ~]# cat /etc/redhat-release 
CentOS Linux release 7.8.2003 (Core)
[root@mail ~]# uname -a
Linux mail.sabonandmore.nl 3.10.0-1127.19.1.el7.x86_64 #1 SMP Tue Aug 25 17:23:54 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
[root@mail ~]# rpm -qa | egrep -i openlitespeed
openlitespeed-1.6.16-1.el7.x86_64
[root@mail ~]# 
```

**The problem:**
Everytime that I gracefully restart the openlitespeed service, I get this in the /usr/local/lsws/logs/stderr.log log file

```bash
sh: usermod: command not found
```

**Comment by kam821** [comment]
It seems to me that the problem may be due to an incomplete PATH variable.
Please print output of commands:
- which usermod
- echo $PATH

If first command output would be empty, try switch user not by 'su root', but 'su - root' and try again, ofc if you are using first command.

**Comment by litespeedtech** [comment]
Did you do graceful restart on the webAdmin? Or use a command?

**Comment by franciscopaniskaseker** [comment]
@litespeedtech graceful restart from webadmin (openlitespeed). Using systemctl it does not happen.

**Comment by litespeedtech** [comment]
I tested with centos 7.6 and 8.2, both have the command "usermod".
You may need to check and install it if not exist.
Our code need it to send the restart command.
Thanks
David

**Comment by franciscopaniskaseker** [comment]
@litespeedtech  I am talking about openlitespeed. The problem continue. The usermod command exist and I can manually execute. I think some OLS script is not with right PATH to the binary.

**Comment by franciscopaniskaseker** [comment]
Executing this command
```bash
/usr/local/lsws/bin/lswsctrl restart
```

I get the problem

```bash
sh: usermod: command not found
```

Now I am using recent released openlitespeed 1.6.18. And the problem continue.
